circuits/lib/utils/poseidon.util.circom

```diff
@@ -0,0 +1,47 @@
+pragma circom 2.0.0;
+
+include "../../../node_modules/circomlib/circuits/poseidon.circom";
+
+// Because of the way the Poseidon hash function is implemented for Solidity, the max number of inputs must be 6
+template PoseidonUtil(valueArraySize) {
```

Comment:
`PoseidonUtil` looks too general and far from the purpose. `ValueArrayHash` or something similar may be a better choice